src/components/select-project/selectProject.html

```diff
@@ -12,7 +12,8 @@
       <ui-select-match>
         {{$select.selected | displayName}}
       </ui-select-match>
-      <ui-select-choices repeat="project in $ctrl.projects | searchProjects : $select.search track by (project | uid)">
+      <ui-select-choices repeat="project in $ctrl.projects | searchProjects : $select.search track by (project | uid)"
+                         group-by="$ctrl.groupChoicesBy">
```

Comment:
Can you add a placeholder to fix #398 while in the code?

Reply:
@spadgett is this good for the placeholder: "Select or Create a Project"?

Reply:
Works for me